matguide.py

```diff
@@ -52,23 +58,23 @@ def __init__(self,  parent=None):
                 exec(str("self.mat"+str(i+1)+"Btn.setText(' -  ')"))
                 exec(str("self.mat"+str(i+1)+"Btn.setEnabled(False)"))
 
-        self.connect(self.mat1Btn, SIGNAL("clicked()"), self.getMat)
-        self.connect(self.mat2Btn, SIGNAL("clicked()"), self.getMat)
-        self.connect(self.mat3Btn, SIGNAL("clicked()"), self.getMat)
-        self.connect(self.mat4Btn, SIGNAL("clicked()"), self.getMat)
-        self.connect(self.mat5Btn, SIGNAL("clicked()"), self.getMat)
-        self.connect(self.mat6Btn, SIGNAL("clicked()"), self.getMat)
-        self.connect(self.mat7Btn, SIGNAL("clicked()"), self.getMat)
-        self.connect(self.mat8Btn, SIGNAL("clicked()"), self.getMat)
-        self.connect(self.nextBtn,  SIGNAL("clicked()"), self.getNext)
-        self.connect(self.prevBtn,  SIGNAL("clicked()"), self.getPrev)
-        self.connect(self.exitBtn, SIGNAL("clicked()"),self.goExit)
-        self.connect(self.maleBtn, SIGNAL("clicked()"),self.getMat)
-        self.connect(self.femaleBtn, SIGNAL("clicked()"),self.getMat)
+        self.mat1Btn.clicked.connect(self.getMat)
+        self.mat2Btn.clicked.connect(self.getMat)
+        self.mat3Btn.clicked.connect(self.getMat)
+        self.mat4Btn.clicked.connect(self.getMat)
+        self.mat5Btn.clicked.connect(self.getMat)
+        self.mat6Btn.clicked.connect(self.getMat)
+        self.mat7Btn.clicked.connect(self.getMat)
+        self.mat8Btn.clicked.connect(self.getMat)
+        self.nextBtn.clicked.connect(self.getNext)
+        self.prevBtn.clicked.connect(self.getPrev)
+        self.exitBtn.clicked.connect(self.goExit)
+        self.maleBtn.clicked.connect(self.getMat)
+        self.femaleBtn.clicked.connect(self.getMat)
 
         screen=QDesktopWidget().screenGeometry()
```

Comment:
Getting error Unresolved reference 'QDesktopWidget'. Not sure what this should reference instead.